src/watcher.ts

```diff
@@ -151,7 +151,14 @@ export class Watcher {
       for (const failure of analysisResult.error) {
         this.#logger.log(failure);
       }
-      process.exit(1);
+      // TODO: it would be good to get all the package.json files that
```

Comment:
There's a TODO right above here "Add support for recovering from..." that this comment supersedes.